### PR TITLE
Adding commas at line ends

### PR DESCRIPTION
**Pull request by ManuelFeller**
As JSON can not work without the , at the end of a key-value pair ;)

### CODE REVIEW

File: examples/devcontainer.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  // Do not forget to remove all comments when you use this example file as JSON does not allow to have comments
```
Comment (tbugfinder) [inline examples/devcontainer.json:2]: hm, maybe we should offload comments to a README.md instead?

Comment (ManuelFeller) [reply examples/devcontainer.json:2]: That could also work, I will make a suggestion

Comment (ManuelFeller) [reply examples/devcontainer.json]: I have moved the comments into a `README.md` just for the examples directory

**Review by tbugfinder — APPROVED** [review]
LGTM! ThankYou
@ManuelFeller

**Pull request merged**